cardano-api/src/Cardano/Api/Block.hs

```diff
@@ -336,6 +329,14 @@ data ChainTip = ChainTipAtGenesis
               | ChainTip !SlotNo !(Hash BlockHeader) !BlockNo
   deriving (Eq, Show)
 
+instance ToJSON ChainTip where
+  toJSON ChainTipAtGenesis = Aeson.String "Tip is currently at genesis block"
+  toJSON (ChainTip slot headerHash (Consensus.BlockNo bNum)) =
+    object [ "slotNo" .= slot
+           , "headerHash" .= Text.decodeUtf8 (serialiseToRawBytesHex headerHash)
+           , "blockNo" .= bNum
+           ]
```

Comment:
Oh, I commented on an older version of this code where it was introduced in an earlier patch. See the comments there: we should be making machine readable JSON instances, not pretending JSON is some pretty-printing human format.